src/models/train.py

```diff
@@ -41,6 +41,8 @@ def add_task_args(parser,name):
 def add_general_args(parent_parser):
     """ Adds arguments that aren't part of pl.Trainer, but are useful
         (e.g.) --no_wandb """
+    parent_parser.add_argument("--optimize", type=str, default="roc_auc",
```

Comment:
Can we call this something like `--checkpoint_metric`? I'm worried this could be confused for an arg for specifying a loss function